### PR TITLE
extend irec claim data

### PR DESCRIPTION
**Pull request by soanvig**

### CODE REVIEW

File: common/config/rush/command-line.json [hunk 1]
```diff
@@ -116,7 +116,7 @@
             "summary": "Runs the full Origin development suite with a local blockchain",
             "safeForSimultaneousRushProcesses": false,
             "autoinstallerName": "rush-tools",
-            "shellCommand": "concurrently --restart-tries 3 --restart-after 1500 -n ganache,backend,ui \"yarn run:ganache\" \"wait-on tcp:8545 && yarn migrate:demo && yarn run:backend\" \"wait-on tcp:3030 && yarn run:ui\"",
+            "shellCommand": "concurrently --restart-tries 3 --restart-after 1500 -n backend,ui  \"wait-on tcp:8545 && yarn migrate:demo && yarn run:backend\" \"wait-on tcp:3030 && yarn run:ui\"",
```
Comment (soanvig) [inline common/config/rush/command-line.json]: todo

File: package.json [hunk 2]
```diff
@@ -12,7 +12,7 @@
         "clean": "lerna run clean --parallel",
         "build:static": "lerna run build:static --parallel",
         "build:ts": "lerna run build:ts --stream",
-        "migrate:demo": "lerna run typeorm:drop --scope @energyweb/exchange && lerna run typeorm:run && lerna run start:defaultConfigs --scope @energyweb/migrations-irec --stream",
+        "migrate:demo": "lerna run typeorm:run && lerna run start:defaultConfigs --scope @energyweb/migrations-irec --stream",
```
Comment (soanvig) [inline package.json]: todo

File: packages/organizations/origin-organization-irec-api/src/irec/irec.service.ts [hunk 3]
```diff
@@ -435,23 +439,44 @@ export class IrecService implements IIrecService {
...
 
+        console.log('c');
+
         const userTradeAccount = fromTradeAccount || (await this.getTradeAccountCode(user));
+        console.log('d', userTradeAccount);
         const accountTo = toRedemptionAccount || (await this.getRedemptionAccountCode(user));
-
+        console.log('e', accountTo);
         const items = await userClient.account.getItems(userTradeAccount);
+        console.log('f');
```
Comment (patryk-kochanski) [inline packages/organizations/origin-organization-irec-api/src/irec/irec.service.ts]: console.log('g');

File: packages/traceability/issuer-irec-api/test/certification-request.e2e-spec.ts [hunk 4]
```diff
@@ -46,7 +46,7 @@ describe('Certification Request tests', () => {
     });
 
     after(async () => {
-        await databaseService.cleanUp();
+        // await databaseService.cleanUp();
```
Comment (patryk-kochanski) [inline packages/traceability/issuer-irec-api/test/certification-request.e2e-spec.ts]: remove or uncomment

**Review by patryk-kochanski — CHANGES_REQUESTED** [review]

File: packages/organizations/origin-organization-irec-api/src/irec/irec.service.ts (hunk 3 above)
Comment (soanvig) [reply packages/organizations/origin-organization-irec-api/src/irec/irec.service.ts]: done

File: packages/traceability/issuer-irec-api/test/certification-request.e2e-spec.ts (hunk 4 above)
Comment (soanvig) [reply packages/traceability/issuer-irec-api/test/certification-request.e2e-spec.ts]: done

**Review by patryk-kochanski — APPROVED** [review]

**Pull request merged**